Source/Methods/RFmxDPD/RFmxDPD.cs

```diff
@@ -97,15 +168,72 @@ public struct MemoryPolynomialResults
         #endregion
 
         #region ConfigureDPD
+        public static Waveform ConfigurePreDPDCrestFactorReduction(RFmxSpecAnMX specAn, Waveform referenceWaveform, PreDpdCrestFactorReduction preDpdCfr, string selectorString = "")

```

Comment:
Sorry to be a stickler, but this is the first time DPD is all caps.

ConfigurePreDpdCrestFactorReduction would better match everything else.

Reply:
6eb090a0facfdb52295d84dd783f3472f2487408